content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -627,13 +631,100 @@ read-modify-write and/or patch are the following:
 It is strongly recommended for controllers to always "force" conflicts, since they
 might not be able to resolve or act on these conflicts.
 
+### Transferring Ownership
+
+In addition to the concurrency controls provided by [conflict
+resolution](#conflicts), Server Side Apply provides ways to perform coordinated
+field ownership transfers from users to controllers.
+
+First, the user delegates ownership of the field to a temporary, uniquely named,
+field manager. Next, the user signals to the controller that it should take
+ownership of the field, usually by setting some other field (perhaps in some
+other object) that the controller observes. Last, controller then claims ownership
+of the field by updating the object or applying a configuration containing the
+field.
+
+Transferring fields to controllers this way ensures the field is not accidently
+removed from the live object, or reset to a default value, at any point of time
+during the transfer.
+
+This approach can be used, for example, to safely transfer ownership of the
+`replicas` field from a user to a controller while enabling Horizontal Pod
+Autoscaling on an object.
+
+Say a user has defined deployment with `replicas` set to 100:
+
+{{< codenew file="application/ssa/nginx-deployment.yaml" >}}
+
+And the user created the deployment using server side apply like so:
+
+```shell
+kubectl apply -f application/ssa/nginx-deployment.yaml --server-side
+```
+
+Before the Horizontal Pod Autoscaler takes over the field and before the user
```

Comment:
Hm, this still isn't quite right, other changes are actually safe, it's removing the replicas field that is unsafe. How about:

Now, the user would like to remove spec.replicas from their manifest, so they don't accidentally fight with the HPA controller. However, there is a race: it might take some time before HPA feels the need to adjust spec.replicas, and if the user removes spec.replicas before HPA writes to the field and becomes its owner, then apiserver will set spec.replicas to 1, its default value. This is not what the user wants to happen, even temporarily.

There are two solutions:

1. (easy) Leave spec.replicas in the manifest; when HPA eventually writes to that field, the system gives the user a conflict over it. At that point, it is safe to remove from the manifest.
2. (more advanced) If, however, the user doesn't want to wait, for example because they want to keep the cluster legible to coworkers, then they can take the following step to make it safe to remove spec.replicas from their manifest:

Reply:
Great. I've added this.